How to/TextSearch/index.js

```diff
@@ -0,0 +1,17 @@
+var pdfviewer = new ej.pdfviewer.PdfViewer({
+    documentPath: 'https://cdn.syncfusion.com/content/pdf/pdf-succinctly.pdf',
+    // resourceUrl:'https://cdn.syncfusion.com/ej2/24.2.3/dist/ej2-pdfviewer-lib'
```

Comment:
check and revert, also use the latest version.

Reply:
Updated and used the latest version